cmake/cplex_interface/CMakeLists.txt

```diff
@@ -5,8 +5,46 @@ project(cplex_interface_test)
 find_package (Threads REQUIRED)
 
 set (CMAKE_CXX_STANDARD 11)
-#This needs to be set to your CPLEX library PATH
-set( CPLEX_PATH /opt/ibm/ILOG/CPLEX_Studio1271 )
+
+# By default, the AUTO_CPLEX_PATH variable is set to true. This means that your CPLEX library
+# will founded automatically. However, your CPLEX library must be located in /opt/ibm/ILOG CPLEX.
+
+#If you want to set your custom CPLEX libraty path, set to "false" the following line:
+set(AUTO_CPLEX_PATH true)  # Ex: set(AUTO_CPLEX_PATH false)
```

Comment:
No idea if this works well, but if does it's a very nice addition! Thanks.
This can later be added to all projects that depend on CPLEX
Similar to
https://github.com/dqrobotics/cpp-examples/blob/master/cmake/dqrobotics_dependencies.cmake